docs/Tutorials/Quick-Start.md

```diff
@@ -60,7 +60,7 @@ $this->hasOne('Avatars', [
 	'className' => 'Burzum/FileStorage.FileStorage',
 	'foreignKey' => 'foreign_key',
 	'conditions' => [
-		'Avatars.model' => 'Avatar'
+		'Avatars.model' => 'Users'
```

Comment:
I would prefer to stick to avatars here. What if you have tow different files or images (Avatar + Company Logo for example) attached to the user? I usually name the model value after the assoc, not the model it is associated to. Or even better: UsersAvatar and UsersCompany for example.